src/SignalR/common/Http.Connections/src/Internal/HttpConnectionDispatcher.cs

```diff
@@ -115,6 +115,9 @@ public async Task ExecuteNegotiateAsync(HttpContext context, HttpConnectionDispa
 
         private async Task ExecuteAsync(HttpContext context, ConnectionDelegate connectionDelegate, HttpConnectionDispatcherOptions options, ConnectionLogScope logScope)
         {
+            // set a tag to allow Application Performance Management tools to differentiate long running requests for reporting purposes
```

Comment:
I haven't seen APM written out like this before

Reply:
I didn't know what it stood for and in 1 year I definitely wont remember.